src/backend/funder.ts

```diff
@@ -2,6 +2,13 @@ import axios from 'axios'
 import { checkAccountStatus, getIsAccountIdOnboarded } from './near'
 import { capsuleServer, sufficientFunds } from './utilities/config'
 
+export async function requestOTP(phoneNumber: string) {
```

Comment:
You should also add a `captchaRes` to `requestOTP()` function. Basically, when phone number toggle is on, we want the user to solve captcha when request for an OTP from their phone number (ie. `requestOTP()` function)